src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/gezagvraag/ErkenningNa01012023.java

```diff
@@ -31,35 +32,46 @@ public void perform() {
         Persoonslijst plPersoon = gezagsBepaling.getPlPersoon();
         Ouder1 persoonOuder1 = plPersoon.getOuder1();
         Ouder2 persoonOuder2 = plPersoon.getOuder2();
+
+        boolean persoonErkend = plPersoon.ongeborenVruchtErkendOfGerechtelijkeVaststelling();
+        boolean persoonOngeborenVruchtErkend = plPersoon.ongeborenVruchtErkend();
+        boolean isPersoonErkend = persoonErkend || persoonOngeborenVruchtErkend;
+
+        if (requirementsForRuleAreMet(persoonOuder1, persoonOuder2, isPersoonErkend)) {
```

Comment:
Early return / guard classes to reduce cognitive complexity.

https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html
https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement